physutils/tests/utils.py

```diff
@@ -77,3 +80,82 @@ def filter_physio(data, cutoffs, method, *, order=3):
     filtered = physio.new_physio_like(data, signal.filtfilt(b, a, data))
 
     return filtered
+
+
+def create_random_bids_structure(data_dir):
+
+    dataset_description = {
+        "Name": "Example BIDS Dataset",
+        "BIDSVersion": "1.7.0",
+        "License": "",
+        "Authors": ["Author1", "Author2"],
+        "Acknowledgements": "",
+        "HowToAcknowledge": "",
+        "Funding": "",
+        "ReferencesAndLinks": "",
+        "DatasetDOI": "",
+    }
+
+    physio_json = {
+        "SamplingFrequency": 10000.0,
+        "StartTime": -3,
+        "Columns": [
+            "time",
+            "respiratory_chest",
+            "trigger",
+            "cardiac",
+            "respiratory_CO2",
+            "respiratory_O2",
+        ],
+    }
+
+    # Create BIDS structure directory
+    subject_id = "01"
+    session_id = "01"
+    task_id = "rest"
+    run_id = "01"
+
+    bids_dir = pjoin(
+        data_dir, "bids-dir", f"sub-{subject_id}", f"ses-{session_id}", "func"
```

Comment:
I think we should make sure that the test covers the possibility to have the recording entity mentioned in the filename. Ex. `sub-006_ses-03_task-test_run-01_recording-cardiac_physio.tsv.gz` would be a valid filename according to the BIDS standard. And since we also included a `recording` parameter in the `load_from_bids` function

Reply:
Sorry @maestroque I think my comment was not clear. The tests should cover the possibility to have filenames with the `recording` entity as well as filenames that does not include that entity.